### DISCUSSION TITLE
Editor complete difficulty adjustment during beatmap

### DISCUSSION THREAD

**Discussion by Ezius** [ppy/osu#14466] [Ideas]
Peppy already mentioned the possibility of changing to different Circle Sizes during a beatmap but I think it should also be an option to also change OD, AR, HP Drain and also Stack leniency and also do stuff like changing the AR, OD, Circle Size or HP drain up to 11 by default instead of 10 (except circles which is 7). 
The Community can then decide what should be rankable and what not and that option would give Mappers a lot of freedom with the Beatmap.

**Comment by frenzibyte** [comment]
The other proposal should definitely be in its own post, given that it's completely irrelevant from the first proposal.

**Reply by Ezius** [reply]
thanks for the info!

**Comment by peppy** [comment]
The hold circle concept is already something on my personal radar (drawing from osu!stream).

**Reply by Ezius** [reply]
okay then I'll just reduce the post to the first proposal thanks! and I also don't need to do another post since it is already on your radar.
Also thanks for being such a nice dude that is this dedicated to this game and the fanbase, it's really hard to actually put it into words.
